docker/docker-compose.lwd.yml

```diff
@@ -0,0 +1,61 @@
+version: "3.8"
+
+services:
+  zebra:
+    ports:
+      # Zebra uses the following inbound and outbound TCP ports
+      - "8233:8233"
+      - "8232:8232"
```

Comment:
These are zebra ports, but this file actually describes the lightwalletd service.

Reply:
Same as https://github.com/ZcashFoundation/zebra/pull/8215#discussion_r1473431847